compiler/src/formatting/format.re

```diff
@@ -24,8 +24,9 @@ type iterator_item_type =
   | IteratedRecordData
   | IteratedValueBindings;
 type expression_parent_type =
-  | GroupedExpression
-  | GenericExpression;
+  | GroupedInfixExpression
+  | GenericExpression
+  | GroupedAccessExpression;
```

Comment:
After reading it now, since it's the parent, I think it makes more sense to drop the `Grouped` part so it becomes just `InfixExpression` and `AccessExpression` 👍